docusaurus.config.js

```diff
@@ -38,7 +38,7 @@ const config = {
 
   scripts: [
     {
-      src: '/lisk-documentation/js/matomo.js',
+      src: '/js/matomo.js',
```

Comment:
probably you need to use append `process.env.LISK_DOC_BASE_URL`?

Reply:
Hmm, but we do not really want Matomo to track on staging anyway

Reply:
i guess that's true, but it seems it's wrong 🤔 also number in the staging probably doesn't affect much anyway

Reply:
> but it seems it's wrong
what is wrong exactly?

```suggestion
      src: process.env.LISK_DOC_BASE_URL + '/js/matomo.js',
```

you mean update it like this? Imho we should only do that if tracking on github pages is blocked. No reason to pollute the tracking results unnecessary. I think it should be blocked by default, but I saw something got tracked on matomo anyway. So if we make this change we should double check. 

But imho we can just leave as is, and there should be no problem, as it works on production, but not on staging, like intended.

Reply:
yea I meant something like that. to do it correctly, we can ignore from certain URL on the matomo side.

Reply:
@shuse2 made the change, please check!